src/components/login-form.tsx

```diff
@@ -71,9 +42,16 @@ export const LoginForm = (): React.ReactElement => {
           errors={passwordErrors}
         />
       </div>
-      <div>
-        <SubmitButton label='Log in' />
-      </div>
+      {loading ? (
+        <div>
+          <SubmitButton label='Loading...' disabled />
+        </div>
+      ) : (
+        <div>
+          <SubmitButton label='Log in' />
+        </div>
+      )}
```

Comment:
Dá pra passar a condicional pro nível mais baixo:

```suggestion
      <div>
        {loading ? <SubmitButton label='Loading...' disabled /> : <SubmitButton label='Log in' />)}
      </div>
```